stdnum/vatin.py

```diff
@@ -44,7 +44,7 @@
 >>> validate('XX')
 Traceback (most recent call last):
     ...
-InvalidComponent: ...
+ImportError: ...
```

Comment:
The `validate()` function should only raise one of the exceptions listed in `stdnum.exceptions`.